jwst/wfs_combine/tests/test_wfs_combine.py

```diff
@@ -89,8 +111,157 @@ def test_create_combined(_jail, wfs_association,
         im1.save(path1)
         im2.save(path2)
 
-    result = WfsCombineStep.call(path_asn, do_refine=False)
+    result = WfsCombineStep.call(path_asn, do_refine=False, flip_dithers=False, psf_size=50,
+                                 blur_size=10, n_size=2)
 
     # Check that results are as expected
     assert result.data[5, 5] == result_data
     assert result.dq[5, 5] == result_dq
+
+
+def test_shift_order_no_refine_no_flip(wfs_association):
+    path_asn, path1, path2 = wfs_association
+    nircam_pixel_size = 0.031 / 3600.
+    delta_pixel = 5
+    with datamodels.open(path2) as im2:
+        im2.meta.wcsinfo = {
+            'dec_ref': 11.99875540218638,
+            'ra_ref': 22.02351763251896 + delta_pixel * nircam_pixel_size,
+            'roll_ref': 0.005076934167039675, 'v2_ref': 86.039011, 'v3_ref': -493.385704,
+            'v3yangle': -0.07385127, 'vparity': -1, 'wcsaxes': 2}
+        im2 = AssignWcsStep.call(im2, sip_approx=False)
+
+        im2.save(path2)
+    wfs = wfs_combine.DataSet(path1, path2, 'outfile.fits', do_refine=False, flip_dithers=False, psf_size=50,
+                              blur_size=10, n_size=2)
+    wfs.do_all
+    assert wfs.input_1.meta.observation.exposure_number == '1'
+    assert wfs.input_2.meta.observation.exposure_number == '2'
+    assert wfs.off_x == abs(delta_pixel)
+    assert wfs.off_y == 0
+    wfs.input_1.close()
+    wfs.input_2.close()
```

Comment:
Ideally `DataSet` class would have an `__enter__` and `__exit__` method each defined so that one could do 

```
with wfs_combine.DataSet(path1, path2, ...):
    do things
```

with no calls to `.close()` needed.  Because of course if any of these asserts fail, then the files get left open.

Reply:
I agree that __enter__ and __exit__ are a more symmetrical way to go but I don't know that it is worth the effort. I worry about unintended consequences. This code will only run during commissioning and it doesn't seem to be worth the effort.
I don't see major consequences of tests failing and thus files being left open. 
If I need to, I'll make the changes.

Reply:
One thing: I'm wrong about the code only running during commissioning. It will run whenever we do a wavefront sensing observation. The current plan is this will happen every two weeks. Even so, the orphan open files would only happen when running tests that fail.